nav2_map_server/test/map_saver_3d_params.yaml

```diff
@@ -0,0 +1,3 @@
+map_saver:
+  ros__parameters:
+    save_map_timeout: 5000.0
```

Comment:
Am I correct, that save map timeout is `5000 seconds` or not???

Reply:
Okk, it was set before in `ms`, with new changes this is changed to `seconds`. 
> 5000.0 -> 5.0